db/migrate/20170712100444_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :comments do |t|
+      t.string :commenter
```

Comment:
seperate model bna user
user will be commenter
matlab a comment belongs to user and user can has many comments